src/DotLiquid/Liquid.cs

```diff
@@ -51,6 +51,7 @@ static Liquid()
             Template.RegisterTag<Tags.Literal>("literal");
             Template.RegisterTag<Tags.Unless>("unless");
             Template.RegisterTag<Tags.Raw>("raw");
+            Template.RegisterTag<Tags.RegisterFilters>("register_filters");

```

Comment:
This would be the first snake case tag. I see IfChanged is not snake case - and that is a standard tag apparently. It doesn't seem like registerfilters should be snake case.

Reply:
Renamed to `addfilters` to prevent conflation with Template.RegisterFilters()